bitnami/mysql/values-production.yaml

```diff
@@ -9,7 +9,7 @@ image:
   ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
   ##
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
why? The tag used is still a rolling tag no?

Reply:
These **values** are intended for production. You can check other **values-production.yaml* files (Redis, MariaDB, etc)